rdmd.d

```diff
@@ -495,10 +581,25 @@ private int rebuild(string root, string fullExe,
             ~ [ "-of" ~ fullExeTemp ]
             ~ [ "-od" ~ objDir ]
             ~ [ "-I" ~ dirName(root) ]
+            ~ (useDashI ? [ "-i", "-X", "-Xf=" ~ depsFilename ] : null)
```

Comment:
will this work with `rdmd -i=std,core main.d` ? or will the added -i (which means -i=-std,-core,-etc,-object) negate the `-i=std,core` ? It looks like it works (ie as the user wanted, with -i=std,core), but I don't understand why, since -i appears after compilerFlags and (IIRC), -i flags are processed left to right (actually docs in dmd --help are not clear on what happens on conflicting -i flags)

Reply:
If the same pattern appears multiple times, the first one has priority (the left-most).

In reality, `-std,-core,-etc,-object` are always present, it's just that they are always added to the end of the list and have the lowest priority.  So if the user specifies any one of them, it will override the default.